test-components/sqlite/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "sqlite"
+name = "sqlite-test-component"
```

Comment:
I'm confused - these test-component crates are not in the same workspace as everything else. Why would this cause an issue?